### PR TITLE
[UXE-3944] fix: add initiatorType field in edge functions

### PR DESCRIPTION
**Pull request by pauloSF0**
### Explain what was fixed and the correct behavior.
fix: add initiatorType field in edge functions
### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/110847590/86ae73a6-bd9f-413c-99f3-518094aad748

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] bug: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [ ] New tests are added to prevent the same issue from happening again
- [ ] UI changes are validated by a team designer
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by HerbertJulio — DISMISSED** [review]

File: src/views/EdgeFunctions/FormFields/FormFieldsCreateEdgeFunctions.vue [hunk 1]
```diff
@@ -49,6 +51,19 @@
     emit('update:previewData', previewValues)
     return previewValues
   })
+
+  const radioOptions = ref([
```
Comment (matheus-sunderhus) [inline src/views/EdgeFunctions/FormFields/FormFieldsCreateEdgeFunctions.vue:55]: sugestão, trocar para initiatorTypeOptions dando contexto a variável e onde ela é usada.

File: src/views/EdgeFunctions/FormFields/FormFieldsEditEdgeFunctions.vue [hunk 2]
```diff
@@ -70,6 +72,19 @@
     emit('update:previewData', previewValues)
     return previewValues
   })
+
+  const radioOptions = ref([
```
Comment (matheus-sunderhus) [inline src/views/EdgeFunctions/FormFields/FormFieldsEditEdgeFunctions.vue:76]: O mesmo aqui.

**Review by matheus-sunderhus — COMMENTED** [review]
minor changes.

File: src/views/EdgeFunctions/FormFields/FormFieldsCreateEdgeFunctions.vue [hunk 3]
```diff
@@ -49,6 +51,19 @@
     emit('update:previewData', previewValues)
     return previewValues
   })
+
+  const initiatorTypeOptions = ref([
```
Comment (william-tome) [inline src/views/EdgeFunctions/FormFields/FormFieldsCreateEdgeFunctions.vue:55]: Isso precisa ser uma ref?? Não me parece que precisa de reatividade e mesmo que pouco, custa mais memória que uma constante.

**Review by william-tome — APPROVED** [review]

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**